apps/boss/index.ts

```diff
@@ -1,12 +1,17 @@
 import './lib/util/UlixeeLogger';
 import '@ulixee/commons/lib/SourceMapSupport';
-import { app } from 'electron';
+import debug from 'debug';
 import log from 'electron-log';
 import { Menubar } from './lib/Menubar';
 
 Object.assign(console, log.functions);
-
-if (app.isPackaged) process.env.DEBUG = 'ulixee:*';
+debug.log = log.debug.bind(log);
+debug.enable('ulixee:*');
+debug.formatArgs = function (args) {
```

Comment:
There are a number of lint issues.

Reply:
fixed this one. Other one is modified in the other pr